doc/cutting-a-release.md

```diff
@@ -81,41 +89,54 @@ git commit -m"Create ${RELEASE}.x release branch" .
 git push --set-upstream origin ${RELEASE}.x
 ```
 
+NOTE: No code review or Pull Request is needed as part of this step.
```

Comment:
Do you want that to be `**NOTE:**` so it is in **BOLD** ?

Reply:
yep. I like that. Done.